files/rh-docker/help.md

```diff
@@ -0,0 +1,72 @@
+% NEXUS(1) Container Image Pages
+% Sonatype
+% February 2, 2020
+
+# NAME
+nexus \- Nexus IQ Server container image
```

Comment:
I'd assume "nexus" standing alone is probably the repo manager. Do we want to update these to "nexus-iq" or something like that?

Reply:
Good point, will do!

Reply:
Just remove it - we're repeating 'Nexus anyhow'...